### PR TITLE
Mavenization, junit 4.12 is added, some refactoring.

### PR DESCRIPTION
**Pull request by ravingbool**

### CODE REVIEW

**Comment by ravingbool** [comment]
Please create a public repo called https://github.com/nicolas-raoul/OSSRH-65862 to verify github account ownership.
It is needed due to https://issues.sonatype.org/browse/OSSRH-65862

**Comment by ravingbool** [comment]
can I add some special symbols translation to KanaToRomaji.java?
aMap.put("「", "\"");
aMap.put("」", "\"");
aMap.put("。", ".");

File: README.md [hunk 1]
```diff
@@ -1,12 +1,37 @@
...
+```
+
 ```
 $ ./jakaroma.sh 六本木ヒルズ森タワー
 Roppongi Hiruzu Mori Tawa-
+```
 
+or just
 ```
+java jakaroma-1.0-SNAPSHOT-jar-with-dependencies.jar -jar 六本木ヒルズ森タワー
```
Comment (nicolas-raoul) [inline README.md:31]: Shouldn't it be `java -jar target/jakaroma-1.0.0-SNAPSHOT-jar-with-dependencies.jar  六本木ヒルズ森タワー` instead? Notice the `target` folder and `1.0.0` version number.

File: jakaroma.sh [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 STRING=$1
 
-java -Dfile.encoding=UTF-8 -cp lib/kuromoji-core-1.0-SNAPSHOT.jar:lib/kuromoji-ipadic-1.0-SNAPSHOT.jar:classes fr.free.nrw.jakaroma.Jakaroma $1
+java -Dfile.encoding=UTF-8 jakaroma-1.0-SNAPSHOT-jar-with-dependencies.jar -jar $1
```
Comment (nicolas-raoul) [inline jakaroma.sh:5]: Same (`target` and version number)

**Review by nicolas-raoul — CHANGES_REQUESTED** [review]
Thanks a lot for doing that!
It compiles fine but I noticed minor problems in the script and documentation, would you mind fixing it, so that I can merge?
Cheers :-)

File: README.md (hunk 1 above)
Comment (ravingbool) [reply README.md]: Fixed

File: jakaroma.sh (hunk 2 above)
Comment (ravingbool) [reply jakaroma.sh]: Fixed

**Comment by ravingbool** [comment]
Hello @nicolas-raoul please take a look again, I fixed the issues :)

**Comment by nicolas-raoul** [comment]
Thanks a lot!
Please do not hesitate to fix any other issue, also the unit tests probably need some maintenance :-)

**Comment by ravingbool** [comment]
Thank you for the quick response @nicolas-raoul! Have a tice day!
Max

**Pull request merged**